lib/modules/apostrophe-workflow-pages/index.js

```diff
@@ -222,9 +222,31 @@ module.exports = {
           req.locale = locale;
 
           return async.eachSeries(parked, function(item, callback) {
+            convertToLocalizedSlug(item, locale);
             return self.implementParkOne(req, item, callback);
           }, callback);
 
+          function convertToLocalizedSlug (item, locale) {
+            if (locale) {
+              var liveLocale = workflow.liveify(locale);
+              if (typeof item.slug === 'object') {
+                if (!item.slug[liveLocale] && !item.slug._default) {
+                  throw new Error('apostrophe-workflow: mising default value in localized parked page');
+                }
+                item.slug = item.slug[liveLocale] || item.slug._default;
+                if (item.slug === '/') {
+                  throw new Error('apostrophe-workflow: the homepage cannot have a localized slug');
```

Comment:
This is probably not good enough, as they probably tried to set it to something other than / otherwise they would not have tried to use the feature. Instead check whether parkedId is `home` and reject in that case. There's still a chance they overrode `minimumPark` and used some other `parkedId` for the home page, but it's unlikely and there isn't much we could do in that case.

Reply:
Ok